client/reducers/helloWorldReducer.ts

```diff
@@ -0,0 +1,14 @@
+import { AppActions } from '../types/actions'
+import * as actions from '../types/actionStrings'
+
+const helloWorldReducer = (state = "", action: AppActions): string => {
+    switch (action.type) {
+        case actions.addToHelloWorld:
```

Comment:
is it better to keep these types here instead of they won't take up too much space?